README.adoc

```diff
@@ -34,16 +37,105 @@ include::https://raw.githubusercontent.com/spring-guides/getting-started-macros/
 include::https://raw.githubusercontent.com/spring-guides/getting-started-macros/master/hide-show-sts.adoc[]
 
 
-
 [[initial]]
-== Create a resource controller
+== Create the database
+
+Go to the terminal (Command Prompt (cmd) in Microsoft Windows). Open MySQL Client with a user that can create new users.
+
+For example: On a Linux, use the command
+
+[source, sh]
+----
+$ sudo mysql --password
+----
+
+NOTE: This connects to MySQL as a root, this is NOT THE RECOMMENDED WAY.
+
+Create a new database
+
+[source, mysql]
+----
+mysql> create database db_example; -- Create the new database
+mysql> create user 'springuser'@'localhost' identified by 'ThePassword'; -- Creates the user
+mysql> grant all on db_example.* to 'springuser'@'localhost'; -- Gives all the privileges to the new user on the newly created database
+----
+
+== Create the Spring Application
+
+Add `pom.xml` dependencies
+
+[source, xml, indent=0]
+----
+<!-- JPA Data (We are going to use Repositories, Entities, Hibernate, etc...) -->
+
+<dependency>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-data-jpa</artifactId>
+    <version>1.4.3.RELEASE</version>
+</dependency>
+
+<!-- Use MySQL Connector-J -->
+
+<dependency>
+    <groupId>mysql</groupId>
+    <artifactId>mysql-connector-java</artifactId>
+    <version>5.1.40</version>
+</dependency>
+----
+
+or in Gradle `build.gradle`:
+
+[source, groovy]
+----
+// JPA Data (We are going to use Repositories, Entities, Hibernate, etc...)
+compile group: 'org.springframework.boot', name: 'spring-boot-starter-data-jpa', version: '1.4.3.RELEASE'
+
+// Use MySQL Connector-J
+compile group: 'mysql', name: 'mysql-connector-java', version: '5.1.40'
+----
+
+Spring Boot gives you defaults on all things, the default in database is `H2SQLDB`, so when you want to change this and use any other database you must define the connection attributes in the `application.properties` file
+
+== Create the `application.properties` file
+
+In the sources folder, you create a resource file `src/main/resources/application.properties`
+
+[source, java]
+----
+include::complete/src/main/resources/application.properties[]
+----
 
-Create a new controller for your Spring application:
+Here, `spring.jpa.hibernate.ddl-auto` can be `update`, `create`, `create-drop`, refer to the Hibernate documentation for details.
```

Comment:
I think `none' is also valid (and it's the default)? Probably worth taking a paragraph to explain here because the app will start when the database is empty with `create` but not when the tables are already defined. So typically the user has to start with 'create' and switch to 'none' or 'update'.